addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -493,7 +514,14 @@ export class Rtc {
                 }
                 break;
             case "trackChange": {
-                const { isSelfMuted, isTalking, isSendingVideo, isDeaf } = payload.state;
+                if (Object.values(this.mapTransceiverToMid).length === 0) {
```

Comment:
`this.mapTransceiverToMid` is a map but you are manipulating it as if it's a pojo, `Object.values()` of a map is an empty array.